### PR TITLE
[Feature]: Add to bin / restore /permanently delete Note

### PR DESCRIPTION
**Pull request by munjalSaksham71**
Features Added: 
1. Delete from homepage and it goes to trash bin
2. restore from trash bin
3. Permanently Delete from trashbin

Issue: Still have to reload if you click on delete/ Add to Archive button in homepage - will work on this issue.

### CODE REVIEW

File: src/context/bin-context.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    }
+
+    useEffect(() => {
+        const getBinNotes = async () => {
+            const data = await getDocs(binCollectionRef);
+            setBinNotes(data.docs.map((doc) => ({ ...doc.data(), id: doc.id })))
+          };
+
+          getBinNotes();
+          // eslint-disable-next-line
```
Comment (ShreyashMahajan) [inline src/context/bin-context.js:41]: remove comments

**Review by ShreyashMahajan — APPROVED** [review]
good to merge

File: src/context/bin-context.js (hunk 1 above)
Comment (munjalSaksham71) [reply src/context/bin-context.js:41]: I have added this to remove warning from useEffect.

**Review by JantuDeb — APPROVED** [review]
This is really awesome

**Pull request merged**